src/content/docs/en/guides/client-side-routing.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Client-side routing (experimental)
+description: >-
+  How to enable experimental support for client-side routing in your Astro site.
+i18nReady: true
+---
+
+Support for client-side routing in Astro projects can be enabled behind an experimental flag. Client-side routing in Astro is powered by a new browser API, [view transitions](https://developer.chrome.com/docs/web-platform/view-transitions/) with fallback options for non-supporting browsers.
+
+Using our new view transitions support you can have seamless animations between pages. Astro comes with:
```

Comment:
We just said it's new above, so I would avoid saying it twice, plus the proper noun situation.
```suggestion
Using the View Transitions support you can have seamless animations between pages. Astro comes with:
```